.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - run:
           name: Prepare Environment
           command: |
-              echo 'export CIRCLE_ARTIFACTS="$CIRCLE_WORKING_DIRECTORY/artifacts"' >> $BASH_ENV
+              echo 'export CIRCLE_ARTIFACTS="/home/circleci/project/artifacts"' >> $BASH_ENV
```

Comment:
`~` and `/home/circleci/` is not the same folder so artifacts were stored in the wrong folder